src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -437,6 +464,12 @@ def geop_tasks(geom, errback):
          nlcd_kfactor)
     ]
 
+    if geom.within(DRB):
+        definitions.append(('nlcd_streams',
+                            {'polygon': [geom.geojson],
+                             'vector': streams(geom, drb=True)},
+                            nlcd_streams_drb))
```

Comment:
That's correct. While we prefer to use the DRB streams for calculating LS if they are available, we _always_ use the NHD streams for Stream Bank Erosion, which are calculated here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/mapshed/tasks.py#L137-L142. Thus, we need the NHD streams in all cases, and the DRB streams only in the cases when they are available.